torch/testing/_internal/common_methods_invocations.py

```diff
@@ -290,6 +308,13 @@ def sample_inputs(self, device, dtype, requires_grad=False, **kwargs):
             samples = self.sample_inputs_func(self, device, dtype, requires_grad, **kwargs)
         except TypeError:
             samples = self.sample_inputs_func(self, device, dtype, requires_grad)
+
+        if include_conjugated_inputs:
+            conj_samples = self.conjugated_sample_inputs(samples)
```

Comment:
Nit: "conjugated_sample_inputs" -> "conjugate_sample_inputs" 

since the function performs the operation